lib/clever_tap/client.rb

```diff
@@ -33,7 +33,7 @@ def connection
         configure.call(config)
 
         # NOTE: set adapter only if there isn't one set
-        config.adapter :net_http if config.builder.handlers.empty?
+        config.adapter :net_http if config.builder.adapter.nil?
```

Comment:
Why is this change needed?

Reply:
> Why is this change needed?

After updating the version we get the adapter details in the adapter inside RackBuilder instead of handlers.